### ISSUE TITLE
empty message server crash

### ISSUE THREAD

**Issue by tubeampprofessor** [suspectpart/alfachat#10]
- chat crashes when user sends an empty message
- wasn't this fixed with the last version already?

Error message:

Internal Server Error

The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

**Comment by suspectpart** [comment]
Yep, I know about this one, but haven't fixed it yet. Always forget it... Thanks for reminding me!
